clouddiscovery/tests/provider/vpc/test_diagram.py

```diff
@@ -186,3 +192,74 @@ def test_should_detect_public_and_private_subnet(self):
                 to_node=ResourceDigest(id=PRIVATE_SUBNET, type="aws_subnet"),
             )
         )
+
+    def test_asg_ecs(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715)